### ISSUE TITLE
Allow storing AppLovinSettings asset in any project folder

### ISSUE THREAD

**Issue by imurashka** [AppLovin/AppLovin-MAX-Unity-Plugin#239]
### Feature Description

Hello!

As a developer using the AppLovin Max SDK, I would like to have the flexibility to store the AppLovinSettings asset in any folder within my project, rather than being restricted to the Resources folder. Since AppLovinSettings is a ScriptableObject used only in the Editor, it would be more convenient to detach it from the Resources folder and allow it to be placed anywhere in the project.

Suggested implementation:

Update the AppLovinSettings.Instance property to search for the asset throughout the entire project. If the asset is not found, create it in a default location. Here's a possible implementation:
```c#
public static AppLovinSettings Instance
{
    get
    {
        if (instance == null)
        {
            instance = FindOrCreateInstance();
        }
        return instance;
    }
}

private static AppLovinSettings FindOrCreateInstance()
{
    var assetGUIDs = AssetDatabase.FindAssets($"t:{nameof(AppLovinSettings)}");
    var assets = assetGUIDs.Select(AssetDatabase.GUIDToAssetPath).Select(AssetDatabase.LoadAssetAtPath<AppLovinSettings>).ToArray();

    switch (assets.Length)
    {
        case 0:
            return CreateAndSaveAsset();
        case > 1:
            Debug.LogWarning("There is more than one instance of AppLovinSettings in the project. Using the first one found.");
            break;
    }
    return assets[0];
}

private static AppLovinSettings CreateAndSaveAsset()
{
    const string defaultAssetPath = "Assets/Settings/AppLovinSettings.asset";

    var asset = CreateInstance<AppLovinSettings>();
    AssetDatabase.CreateAsset(asset, defaultAssetPath);
    AssetDatabase.SaveAssets();
    AssetDatabase.Refresh();
    Debug.Log($"AppLovinSettings asset created at: {defaultAssetPath}");
    return asset;
}
```
By removing the dependency on the Resources folder, this change will improve the user experience for developers working with AppLovinSettings in the Unity Editor.

**Comment by imurashka** [comment]
Hello @santoshbagadi! Did you have time to take a look at this feature proposal? We tested it in our company and it works well. It would be great if you will add it to SDK. Or I can create pull request if you want.

**Comment by santoshbagadi** [comment]
@imurashka , this is now available in our latest plugin release  [v5.10.0](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_5_10_0).

**Comment by imurashka** [comment]
@santoshbagadi, thank you so much!
